protocols/bgp/server/update_sender.go

```diff
@@ -383,3 +383,18 @@ func (a *UpdateSender) ReplacePath(*net.Prefix, *route.Path, *route.Path) {
 func (a *UpdateSender) RefreshRoute(*net.Prefix, []*route.Path) {
 
 }
+
+// LPM is here to fulfill an interface
+func (a *UpdateSender) LPM(pfx *net.Prefix) (res []*route.Route) {
+	return nil
```

Comment:
To be consistent with above there should be a warning like

`log.Warningf("BGP Update Sender: RouteCount not implemented")`

Reply:
Done